src/main/webapp/election-info.js

```diff
@@ -256,4 +256,15 @@ function addPollingStationMarker(map, position, title, description) {
   marker.addListener('click', () => {
     infoWindow.open(map, marker);
   });
-}
\ No newline at end of file
+}
+
+/**
+ * Handlebars helper that removes the beginning "http://" and the trailing "/"
+ * from the supplied URL string.
+ * 
+ * @param {String} URL to be stripped.
+ */
+Handlebars.registerHelper('stripUrl', function(urlString){
+  urlString = urlString.replace(/(^\w+:|^)\/\//, '');
+  return urlString.slice(0, -1);
```

Comment:
Please check whether the last character is actually "/" instead of blindly dropping the last character?

Reply:
done